README.md

```diff
@@ -1 +1,27 @@
-# llm_validation
\ No newline at end of file
+## HOW TO RUN:
+
+```
+python -m venv .venv
+.venv\Scripts\activate 
```

Comment:
not working for my machine, Docker must have when working in envirment other then some specific Linux disctribution
<img width="659" alt="Screenshot 2024-08-07 at 17 57 40" src="https://github.com/user-attachments/assets/b599d090-7c5e-4916-b9a8-4d59304bcca7">

Reply:
Actually I just wanted to save synchronous version by commit and finish asynchronous tommorow and then conterize this and so on but ok i will do this with synchronous version

Reply:
as you wish, just ping me when you think this PR is ready for me to try to use, don't want spend much time reviewing something I can't really use